CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - `CadenzaClient#fetchAreaIntersections`
 - `CadenzaClient#fetchObjectInfo`
+- `CadenzaActionEvent`, which is fired as 'action' event when clicking a postMessage action Cadenza link in views shown via `CadenzaClient#show`
```

Comment:
Usually we leave the details to the apidocs to not have it in two places. For example the event is fired also for #showMap, but it's also not mentioned here.